src/main/java/com/epam/springtest/pageobject/NavigationPage.java

```diff
@@ -16,15 +19,34 @@
  * <p>Encapsulates common actions and elements, such as opening the base URL and reading the welcome
  * message.
  */
+@Slf4j
 @Getter
 @Component
 @RequiredArgsConstructor
 public class NavigationPage {
 
   private final AppProperties properties;
-  private final SelenideElement welcomeMessage = $(byTagName("h1"));
+  private final SelenideElement headingMessage = $(byTagName("h1"));
+  private final SelenideElement subHeadingMessage = $(byTagName("h2"));
+  private final SelenideElement footerMessage = $("#page-footer");
+  private final SelenideElement forkMeBadge = $("a").$("img[alt='Fork me on GitHub']");
+  private final ElementsCollection links = $("#content").$$("a");
+  private final String LINK_TEMPLATE = "#content a";
 
   public void openHomePage() {
     open(properties.getBaseUrl());
   }
+
+  public void listPresentLinks() {
```

Comment:
I would suggest to rename it to 
`logPresentLinks` so name would give clear information of what is happening in the method.